server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -367,11 +368,13 @@ private DivTag renderCardApplications(AccountModel account) {
       return div();
     }
 
-    ImmutableList<ApplicationModel> newestApplicantApplications =
-        newestApplicant.get().getApplications();
-    int applicationCount = newestApplicantApplications.size();
+    ImmutableList<ApplicationModel> submittedApplications =
+        newestApplicant.get().getApplications().stream()
+            .filter(application -> application.getLifecycleStage() == LifecycleStage.ACTIVE)
```

Comment:
**suggestion:** You can also use `.equals` instead - `application.getLifecycleStage().equals(LifecycleStage.ACTIVE)`

Reply:
I considered using `.equals`, but found this Stack Overflow discussion and was convinced to use `==` instead:  [discussion](https://stackoverflow.com/questions/1750435/comparing-java-enum-members-or-equals).